components/popup/test/popup.test.js

```diff
@@ -87,4 +88,31 @@ describe('popup', () => {
 				.equal(false);
 		});
 	});
+
+	describe(`anchor`, function () {
+		it(`should not find anchor element on show`, function () {
+			const [actualElement] = addElement(
+				textToDomToParent(`<${COMPONENT_NAME}></${COMPONENT_NAME}>`)
+			);
+			actualElement.anchor = "anchor";
+			actualElement.show();
+			setTimeout(() => {
+				expect(actualElement.open)
+					.to
+					.equal(false);
+			}, DELAY);
+		});
+
+		it(`should not find anchor element on open`, function () {
```

Comment:
Same here. The actual `product` of this use case is the state of the popup (value of open).  Hence, it should be reflected in the description of the `it`.
```suggestion
		it(`should not open the popup if anchor does not exist`, function () {
```